buildSrc/src/main/java/AndroidX.kt

```diff
@@ -1,7 +1,7 @@
 object AndroidX {
```

Comment:
Personal opinion:

I'm not a fan of having different Objects files for each dependency

I rather have just one
"Deps" object, and have everything there